examples/pytorch/language-modeling/run_clm_no_trainer.py

```diff
@@ -597,7 +597,7 @@ def group_texts(examples):
                 outputs = model(**batch)
 
             loss = outputs.loss
-            losses.append(accelerator.gather(loss.repeat(args.per_device_eval_batch_size)))
+            losses.append(accelerator.gather_for_metrics(loss.repeat(args.per_device_eval_batch_size)))
```

Comment:
You then need to remove the truncation at line 603 below :-)

Reply:
fixed.